src/nvim/lua/vim.lua

```diff
@@ -118,11 +118,53 @@ local function _update_package_paths()
   last_nvim_paths = cur_nvim_paths
 end
 
+local function rpcrequest(...)
```

Comment:
should have a `_` prefix, we aren't yet sure where we want to publish the VimL functions for Lua.

    local function _rpcrequest(...)

Reply:
The function is not published, that is controlled by the module below. And single `_` is already used for functions that are published but only intended for internal use by C code. Maybe two `__` for strictly local functions, or perhaps that makes it look too much like python?

Reply:
I'll make it two underscores then.